mailbox/api/src/test/java/org/apache/james/mailbox/model/AttachmentTest.java

```diff
@@ -45,6 +45,18 @@ public void streamShouldBeConsumedOneTime() throws Exception {
     }
 
     @Test
+    public void getByteShouldReturnByteArrayRepresentingTheAttachment() throws Exception {
+        String input = "mystream";
+        Attachment attachment = Attachment.builder()
+            .bytes(input.getBytes())
+            .type("content")
+            .build();
+
+        byte[] bytes = attachment.getBytes();
+        assertThat(new String(bytes)).isEqualTo(input);
```

Comment:
Let's enforce good habits in tests also IMO. Especially when it is easy to apply good habits.